pages/integrations/github.md.erb

```diff
@@ -79,3 +79,68 @@ If you want to run builds *only* on pull requests, set the Branch Filter Pattern
 ## Noreply email handling
 
 When you [connect your GitHub account to Buildkite](#connecting-buildkite-and-github) the email address associated with the GitHub account is added to your Buildkite account. If you've got GitHub set not to display your email, `[username]@users.noreply.github.com` or the more recent `[username+id]@users.noreply.github.com` is added instead. The email address of a commit is one of the ways Buildkite matches webhook builds to users.
+
+## Customizing commit statuses
+
+The commit status is the label used to identify the Buildkite checks on your commits and pull requests. Normally, Buildkite autogenerates these statuses. 
+
+For example, if you select _Update commit statuses_ in your _Pipeline Settings_:
+
+<%= image "update-commit-statuses-on.png", alt: "Screenshot of GitHub build settings with Update commit statuses enabled" %>
+
+Your checks will appear on your pull request as _buildkite/`<pipeline-name>`_:
+
+<%= image "github-default-status.png", alt: "Screenshot of the resulting GitHub pull request statuses" %>
+
+
+You can customize the commit statuses, for example to reuse the same pipeline for multiple components in a monorepo,  at both the build and step level, using the [`notify`](/docs/pipelines/notifications) attribute in your `pipeline.yml`.
+
+### Build level
+
+1. Add the following to your `pipeline.yml`, at the top level:
+
+    ```yaml
+    notify:
+        - github_commit_status:
+            context: "my-custom-status"
+    ```
+2. In _Pipeline Settings_ > _GitHub_, make sure _Update commit statuses_ is not selected. Note that this prevents Buildkite from automatically creating and sending statuses for this pipeline, meaning you will have to handle all commit statuses through the `pipeline.yml`.
+3. When you make a new commit or pull request, you should see _my-custom-status_ as the commit status:
+    <%= image "github-custom-status.png", alt: "Screenshot of GitHub build settings and the resulting GitHub pull request statuses" %>
+
+In a basic setup (one `pipeline.yml` for a repo containing one codebase), this customizes the commit status for the pipeline. However, if you have multiple `pipeline.yml` files in one repo, feeding in to the same Buildkite pipeline, this allows you to have different statuses when building different sections of the repo.
+
+For example, if you have a monorepo containing three applications, you could use the same pipeline, with different `pipeline.yml` files for each application:
+
+```
```

Comment:
Would a screenshot of actual files makes sense?

Reply:
Ah this was a made up example. I can create some files and screenshot them on GitHub if you wish? Although I wouldn't be able to get all the directories and individual yamls into one screenshot . . .

Reply:
You could literally do it on the local file system for this example? Show the same 3 dirs + 3 files but not in plain text?

Reply:
I've seen a thing that converts these diagrams to images, but can't remember what it was.

Reply:
Am I being fabulously daft? How do you simultaneously screenshot three same-level dirs and the files inside them?